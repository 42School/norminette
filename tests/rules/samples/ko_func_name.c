int	BAD_funcname1(void);
int	bad_funcName2(void);
int	B(void);
