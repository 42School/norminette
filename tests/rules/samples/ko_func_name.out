[36mko_func_name.c[0m - [32mIsFuncPrototype[0m In "GlobalScope" from "None" line 1":
		<INT> <TAB> <IDENTIFIER=BAD_funcname1> <LPARENTHESIS> <VOID> <RPARENTHESIS> <SEMI_COLON> <NEWLINE>
[36mko_func_name.c[0m - [32mIsFuncPrototype[0m In "GlobalScope" from "None" line 2":
		<INT> <TAB> <IDENTIFIER=bad_funcName2> <LPARENTHESIS> <VOID> <RPARENTHESIS> <SEMI_COLON> <NEWLINE>
[36mko_func_name.c[0m - [32mIsFuncPrototype[0m In "GlobalScope" from "None" line 3":
		<INT> <TAB> <IDENTIFIER=B> <LPARENTHESIS> <VOID> <RPARENTHESIS> <SEMI_COLON> <NEWLINE>
ko_func_name.c: Error!
Error: INVALID_HEADER       (line:   1, col:   1):	Missing or invalid 42 header
